include/exadg/incompressible_navier_stokes/user_interface/parameters.h

```diff
@@ -56,6 +57,21 @@ class Parameters
   bool
   viscous_problem() const;
 
+  bool
+  viscous_term_is_linear() const;
+
+  bool
+  viscous_term_is_nonlinear() const;
+
+  bool
+  viscosity_is_variable() const;
+
+  bool
+  convective_term_is_treated_implicitly() const;
+
+  bool
+  nonlinear_viscous_term_is_treated_implicitly() const;
```

Comment:
It looks as if some of these member functions are only used inside this class. Then, they should be made private I think. Could you please check this again.

Reply:
that is true, 'viscous_term_is_nonlinear' and 'viscous_term_is_linear' are now private!